test/SpanUtilsTest.cpp

```diff
@@ -0,0 +1,412 @@
+#include "pch.h"
+#include <kf/SpanUtils.h>
+
+SCENARIO("SpanUtils")
+{
+    GIVEN("an array of int for span_cast from int to std::byte")
+    {
+        int arr[] = { 1, 2, 3, 4 };
```

Comment:
Let's have more `constexpr` (and prefix `k`)

Reply:
done